subgraphs/enzyme-core/mappings/v4/ExternalPositionManager.ts

```diff
@@ -2889,6 +2897,177 @@ export function handleCallOnExternalPositionExecutedForFund(event: CallOnExterna
     return;
   }
 
+  if (type.label == 'MYSO_V3_OPTION_WRITING') {
+    if (actionId == MysoV3ActionId.CreateEscrowByTakingQuote) {
+      let decoded = ethereum.decode(
+        '((address,uint48,address,uint48,uint128,uint128,(uint64,address,bool,bool,address)),(uint128,uint256,bytes,address),address)',
+        event.params.actionArgs
```

Comment:
`tuplePrefixBytes` is missing because we have `bytes`
```suggestion
        tuplePrefixBytes(event.params.actionArgs)
```